resources/cdk/README.md

```diff
@@ -4,6 +4,14 @@ This document describes the resources created by the AWS CDK apps in this direct
 
 For instructions on how to run any of these apps, see [Running a CDK app](#cdk).
 
+## rekognition-sns-video-analyzer
+This AWS CDK app creates the following resources:
+- An AWS Identity and Access Management (IAM) unauthenticated role with permissions 
+  for Amazon Simple Email Service (Amazon SQS), Amazon Simple Storage Service (Amazon S3), 
+  and Amazon Rekognition.
+- An Amazon S3 bucket.
```

Comment:
Make sure Amazon S3 is spelled out as Amazon Simple Storage Service on first use.

Reply:
Yes, defined in line above